jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -36,13 +44,62 @@ export class ChromeDevToolsProxyServer {
       ws.send('hello');
     });
 
+    server.listen(SERVER_PORT);
+
+    // TODO: move callback out to standalone method
+    server.on('request', function (request: http.IncomingMessage, response: http.ServerResponse) {
+      var url = request.url || '';
+      if (url == '/json') {
+        url = '/json/list';
+      }
+
+      if (request.method != "GET" || url.slice(0, 6) != '/json/') {
```

Comment:
Let's stick with one type of quotes (single quotes).

Reply:
Use `url.startsWith()` instead of `slice`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

That way you can also get rid of the special casing of `'/json'` in the lines above, and make `url` a `const`

Reply:
There is some special casing because '/json' is an alias for '/json/list'... but I'll try to reduce it in this next patch.